packages/mobile/src/i18n.ts

```diff
@@ -47,34 +44,13 @@ function getAvailableResources() {
 
 const availableResources = getAvailableResources()
 
-function getLanguage() {
-  // We fallback to `undefined` to know we couldn't find the best language
-  // In that case i18n.language will report `undefined` but will use fallbackLng internally
-  const fallback = { languageTag: undefined }
-  const { languageTag } =
-    RNLocalize.findBestAvailableLanguage(Object.keys(availableResources)) || fallback
-  return languageTag
-}
-
-const languageDetector: LanguageDetectorModule = {
-  type: 'languageDetector',
-  detect: getLanguage,
-  init: () => {
-    Logger.debug(TAG, 'Initing language detector')
-  },
-  cacheUserLanguage: (lng: string) => {
-    Logger.debug(TAG, `Skipping user language cache ${lng}`)
-  },
-}
-
-i18n
-  .use(languageDetector)
-  .use(initReactI18next)
-  .init({
+export const initI18n = async (lng: string) => {
```

Comment:
nit: `lng` -> `language` ("language" is used in other variable names)

Reply:
code golf (drop the asyc, see my comment below) and nit (use the function expression because that's the convention):
```
export function initI18n(language: string) {
```

Reply:
good one, i've updated :)